en/hardware/porting_guide.md

```diff
@@ -80,3 +80,24 @@ It is generally recommended to connect RC via separate RX and TX pins to the mic
 If however RX and TX are connected together, the UART has to be put into singlewire mode to prevent any contention. 
 This is done via board config and manifest files. 
 One example is [px4fmu-v5](https://github.com/PX4/Firmware/blob/master/src/drivers/boards/px4fmu-v5/manifest.c).
+
+
+## Accepting a Board into PX4 Codelines
```

Comment:
@hamishwillee 

> Port source code available in the PX4 repository

I do not know how to parse this sentence. 

"Port: is an overloaded term in CS. The act of porting PX4 to new hardware is a "PX4 Port" to XYZ HW or a Port of PX4 to XYZ hardware.

Typically a  Board Support Packages (BSP) is created to facilitate porting. PX4 does not use that term, but in essence that is what it is.

Is this what you are saying: Board Support source code available in the PX4 repository ?

Reply:
@davids5 Actually that was Ramon's text, but I interpret it as "PX4 Port available in the PX4 repository" and have changed appropriately above.

Is it otherwise OK?

@mrpollo And is it OK for you. If so I can merge this into the PR and merge the PR. (Ie what else to happen?)

Note that some of this might need to be moved around/reused in future. The reason being that if we explain base porting in the context of the different types of porting you might want to do (ie what lets you remain compatible) then we may need to explain the benefits of compatibility earlier. We will see.

Reply:
lets merge his 👍